src/modules/portfolio/components/markets/markets.jsx

```diff
@@ -131,6 +135,10 @@ class MyMarkets extends Component {
             paginationPageParam="open"
             collectMarketCreatorFees={p.collectMarketCreatorFees}
             isMobile={p.isMobile}
+            isForking={p.isForking}
+            isForkingMarketFinalized={p.isForkingMarketFinalized}
+            forkingMarket={p.forkingMarket}
+            updateModal={p.updateModal}
           />
```

Comment:
properties not needed here they can be injected at market-properties component container. see above.